api/tests/integration/domain/usecases/get-candidates-import-sheet/get-candidates-import-sheet_test.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+
+const { expect, databaseBuilder } = require('../../../../test-helper');
+
+const readOdsUtils  = require('../../../../../lib/infrastructure/utils/ods/read-ods-utils');
+
+const sessionRepository = require('../../../../../lib/infrastructure/repositories/session-repository');
+const getCandidatesImportSheet = require('../../../../../lib/domain/usecases/get-candidates-import-sheet');
+
+describe('Integration | UseCases | getCandidatesImportSheet', () => {
+
+  const expectedOdsFilePath = `${__dirname}/candidates_import_template.ods`;
+  const actualOdsFilePath = `${__dirname}/candidates_import_template.tmp.ods`;
+
+  let userId;
+  let sessionId;
+
+  beforeEach(async () => {
+    const certificationCenterName = 'Centre de certification';
+    const certificationCenterId = databaseBuilder.factory.buildCertificationCenter({ name: certificationCenterName }).id;
+
+    userId = databaseBuilder.factory.buildUser().id;
+    databaseBuilder.factory.buildCertificationCenterMembership({ userId, certificationCenterId });
+
+    sessionId = databaseBuilder.factory.buildSession({
+      id: 10,
+      certificationCenter: certificationCenterName,
+      certificationCenterId: certificationCenterId,
+      accessCode: 'ABC123DEF',
+      address: '3 rue des bibiches',
+      room: '28D',
+      examiner: 'Johnny',
+      date: '2020-07-05',
+      time: '14:30',
+      description: 'La super description',
+    }).id;
+
+    await databaseBuilder.commit();
+  });
+
+  afterEach(() => {
+    fs.unlinkSync(actualOdsFilePath);
```

Comment:
Eviter les actions synchrones, ça va bloquer Node le temps de la terminer.
```suggestion
    fs.unlink(actualOdsFilePath);
```

Reply:
à faire à part